.github/workflows/dependency-check.yml

```diff
@@ -1,12 +1,23 @@
-name: Go Dependency Checker
+name: Dependency Vulnerability Check
 
 on: [push]
 jobs:
-  build:
+  Go:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
-    - name: Check
-      uses: sonatype-nexus-community/nancy-github-action@76cc24d43c279e8f357d4dccf5ce226835f8385f
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.15
+      id: go
+
+    - name: Write Go Modules list
+      run: go list -json -m all > go.list
+
+    - name: Check vulnerabilities
+      uses: sonatype-nexus-community/nancy-github-action@b492e6567a301a914bd227d44bf10dcfe85438ad
       with:
-        target: go.sum
+        nancyCommand: sleuth -e bba60acb-c7b5-4621-af69-f4085a8301d0,d373dc3f-aa88-483b-b501-20fe5382cc80,4a444740-66e6-4268-9191-c12780af5d4e,5def94e5-b89c-4a94-b9c6-ae0e120784c2
```

Comment:
What are these UUID for?

Reply:
They are OSS Index IDs. I've added a comment on how to get more info as this list changes.